aries-site/src/pages/templates/toast-notifications.mdx

```diff
@@ -15,7 +19,13 @@ import { Box } from 'grommet';
 
 ## Guidance
 
-Everything that you need to know about toasts and more.
+Toasts are a low-attention notification mechanism primarily for use in confirming task receipt and completion.
+
+Toasts are intended to be non-intrusive and brief. A user must be able to read and comprehend a toast message in ~8 seconds before the toast automatically closes itself.
+
+Toasts are not a guaranteed delivery mechanism. They can easily be missed and are never shown while the user is focused on performing a task.
+
+Because there is no guarantee that a user will see a toast and because of the auto-close nature of toasts, they must never be used as a primary means of notification. A separate means (E.g., a notification center) to view active and historical task and alert information is needed.
 
 ### When should you use a Toast Notification?
```

Comment:
In the text that follows:

- I'm not sure what "low attention" means.
- what is a "singular status update"?
- Confirmations could be interpreted as a means of the user confirming something, like a confirmation modal.